### ISSUE TITLE
Uncaught TypeError: Storage.removeItem is not a function

### ISSUE THREAD

**Issue by aubrey-cpt** [tus/tus-js-client#151]
Apologies this is take 2, I wanted to edit the previous logged issue and by mistake closed it.

**Describe the bug**
The error is displayed after a successful upload using Chrome browser.

**To Reproduce**
- Select the file, using file html element in the form.
- Start upload, on successful upload the following error is displayed.
- Uncaught TypeError: Storage.removeItem is not a function at XMLHttpRequest.xhr.onload (tus.js)

**Error source**
File: /lib/upload.js

```js
if (offset == this._size) {
  if (this.options.removeFingerprintOnSuccess && this.options.resume) {
    // Remove stored fingerprint and corresponding endpoint. This causes
    // new upload of the same file must be treated as a different file.
    Storage.removeItem(this._fingerprint);
  }
```

**Fix**

```js
if (offset == this._size) {
  if (this.options.removeFingerprintOnSuccess && this.options.resume) {
    // Remove stored fingerprint and corresponding endpoint. This causes
    // new upload of the same file must be treated as a different file.
    this._storage.removeItem(this._fingerprint);
  }

**Comment by Acconut** [comment]
> Apologies this is take 2, I wanted to edit the previous logged issue and by mistake closed it.

No worries. For next time, it's possible to reopen an issue without problems.

Thank you for reporting this issue and also providing a possible fix. I will address this problem ASAP. The problem likely appeared with https://github.com/tus/tus-js-client/pull/73 (/cc @ifedapoolarewaju) and should only be present in the prelease version 1.8.0-0 and not in the latest stable version 1.7.0.

**Comment by Acconut** [comment]
I addressed the problem and released it as `1.8.0-1`. Could you please give it a try and report back if the problems isn't solved?
